Source/CheckoutFont.swift

```diff
@@ -13,7 +13,7 @@ extension UIFont {
 
 extension UIFont {
 
-    enum GraphikStyle: String, CaseIterable {
+    @frozen enum GraphikStyle: String, CaseIterable {
```

Comment:
It doesn't make sense to add @frozen to something that is not public

Reply:
frozen : Apply this attribute to a structure or enumeration declaration to restrict the kinds of changes you can make to the type. This attribute is allowed only when compiling in library evolution mode. When the compiler isn’t in library evolution mode, all structures and enumerations are implicitly frozen, and this attribute is ignored.